### PR TITLE
Created entitites & Edited started.sql

### PR DESCRIPTION
**Pull request by simionescuraluca**

### CODE REVIEW

File: src/main/java/com/vinyl/model/PurchaseProduct.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		public PurchaseProductId() {
+
+		}
+
+		@Override
+		public int hashCode() {
+			final int prime = 31;
+			int result = 1;
+			result = prime * result + ((product == null) ? 0 : product.hashCode());
+			result = prime * result + ((purchase == null) ? 0 : purchase.hashCode());
```
Comment (nicolaevieru) [inline src/main/java/com/vinyl/model/PurchaseProduct.java:73]: was this duplication intended?

File: src/main/java/com/vinyl/model/Purchase.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	 @ManyToMany
+	 @JoinTable(name="PURCHASE_PRODUCT",
+			 joinColumns=
+			 				@JoinColumn(name="purchase_id", referencedColumnName="id"),
+			 inverseJoinColumns=
+			 				@JoinColumn(name="product_id", referencedColumnName="id")
+	 )
+	 public List<Product> products;
+
+
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/model/Purchase.java:40]: format this better (remove unnecessary empty lines)

Comment (dragosslujeru) [reply src/main/java/com/vinyl/model/Purchase.java:40]: please also check other classes

File: src/main/java/com/vinyl/model/PurchaseProduct.java (hunk 1 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/model/PurchaseProduct.java:73]: no.. it was generated automatically this way

File: src/main/java/com/vinyl/model/Purchase.java (hunk 2 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/model/Purchase.java:40]: done :D

File: src/main/java/com/vinyl/model/Role.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	@NotNull
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Integer id;
+	
+	@NotNull
+	@Column(nullable = false)
+	private String roleName;
+
+
+	public Role(@NotNull String roleName) {
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/model/Role.java]: you now don't have a no args construcor which is needed for this class to be a bean, add that constructor, or remove this one if it is not used

File: src/main/java/com/vinyl/model/PurchaseProduct.java (hunk 1 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/model/PurchaseProduct.java:73]: + one is product, one is purchase

File: src/main/java/com/vinyl/model/Role.java (hunk 3 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/model/Role.java]: done; removed constructor with arguments

**Review by dragosslujeru — APPROVED** [review]

**Pull request merged**